### ISSUE TITLE
Support accessibilityLabel and testId for testing

### ISSUE THREAD

**Issue by Belobobr** [aksonov/react-native-router-flux#3018]
### Version
Tell us which versions you are using: 

- react-native-router-flux v4.0.0.beta-27 
- react-native v0.55.2

### Expected behaviour
I should be able to test using accessibilityLabel and testId

### Actual behaviour
I am not able to test using accessibilityLabel and testId

**Comment by aksonov** [comment]
All props you define as props will be passed to your Scene component. Also you can define it inside your scenes.

**Comment by emiliodacosta** [comment]
"react-native-router-flux": "^4.2.0",
"react-native": "0.62.2",
"appium": "^1.17.1",

Despite this issue being closed, defining accessibilityLabel as a prop within the Scene component does NOT work. Appium is not able to inspect and access an "accessibility id" the way that it does every time that I define it on other Text elements as shown:

**WORKS**

<Text 
        style={styles.greetingText} 
        accessibilityLabel={'TimeOfDayGreetingText'}>
        {this.state.partOfDay} {this.state.firstName}
</Text>

![Screen Shot 2020-06-18 at 12 49 23 PM](https://user-images.githubusercontent.com/27406699/85049589-dbc12100-b162-11ea-9d61-5fe8c3c2520a.png)


**DOES NOT WORK**

<Scene
        key="more"
        component={More}
        title="More"
        accessible
        accessibilityLabel={"More"}
/>

![Screen Shot 2020-06-18 at 12 48 53 PM](https://user-images.githubusercontent.com/27406699/85049599-df54a800-b162-11ea-90bf-33a753300a86.png)

**Comment by emiliodacosta** [comment]
> ### Version
> Tell us which versions you are using:
> 
> * react-native-router-flux v4.0.0.beta-27
> * react-native v0.55.2
> 
> ### Expected behaviour
> I should be able to test using accessibilityLabel and testId
> 
> ### Actual behaviour
> I am not able to test using accessibilityLabel and testId

@Belobobr did it work for you after @aksonov claimed it was fixed

**Comment by anuilkumar-sdei** [comment]
@aksonov  is there any update?

@Belobobr 
@emiliodacosta were you able to solve this or any workaround for same?
